Catch `ValueError` regardless of py version.

mpapierski opened the pull request:
A small adhoc change. 

It turns out on travis sometimes we run rita.py on py2, where JSONDecodeError doesn't exist.

Example failed job: https://travis-ci.org/althea-mesh/althea_rs/jobs/413146414

drozdziak1 commented on integration-tests/rita.py:
```diff
@@ -693,7 +693,9 @@ def test_endpoints_all(self):
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
Unnecessary, the `assert_test` below will print `<red plus> Decoding the JSON Failed`

drozdziak1 commented on integration-tests/rita.py:
```diff
@@ -711,7 +713,9 @@ def test_endpoints_all(self):
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
same here

drozdziak1 commented on integration-tests/rita.py:
```diff
@@ -729,7 +733,9 @@ def test_endpoints_all(self):
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
here

drozdziak1 commented on integration-tests/rita.py:
```diff
@@ -747,7 +753,9 @@ def test_endpoints_all(self):
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
and here

mpapierski commented:
@drozdziak1 As discussed I removed `if VERBOSE` branch and it will report invalid responses unconditionally. Updated with some more changes because because I realized I'll be reporting `Popen` object instead of actual `stdout` output.
